Automattic/Gravatar-SDK-Android
# Align left side of the Gravatar icon with other composables in the pr…

Closes #194

### Description

**Note: I'm targeting another branch that is in a PR state because this PR is also affecting the loading state**

This PR aims to align the left side of the Gravatar icon with other composables in the profile cards (Profile and LargeProfile).
The Gravatar icon has padding so to align it with other views we have to set a negative offset to it.
Additionally, the loading skeleton view was also updated not to have the padding at the start and end of the row. Instead, there's a spacing set only between the items using `Arrangement.spacedBy(4.dp)`.

| Before | After |
|---|---|
| <img width="353" alt="Screenshot 2024-07-02 at 10 32 31" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/c88aef5e-52bc-43f9-89d9-f8f2502e9d03"> | <img width="582" alt="Screenshot 2024-07-01 at 16 42 13" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/a36eae65-6d2b-44f0-895c-1f1c7672b5c0"> |
| <img width="351" alt="Screenshot 2024-07-02 at 10 32 25" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/4e41e7ff-e07f-47d2-a24c-6a7bed366491"> | <img width="357" alt="Screenshot 2024-07-02 at 10 27 31" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/39d0b868-1240-4d5a-8415-467c370ea0a5"> |

There should be some failures in `Profile` and `LargeProfile` screenshots tests. I will record and push new screenshots. 

### Testing Steps

1. Open the Demo app
2. Go to the profile tab
3. Load the profile
4. Enable the layout inspector
5. Confirm the Gravatar icon is aligned with the avatar and other composables that are aligned to the left of the profile card.
